cirq-core/cirq/testing/consistent_act_on.py

```diff
@@ -95,7 +95,7 @@ def assert_all_implemented_act_on_effects_match_unitary(
             )
         return None
 
-    qubits = LineQubit.range(protocols.num_qubits(val) * 2)
+    qubits = LineQubit.range(num_qubits_val * 2)
```

Comment:
[No change needed] I don't see a use case for zero-qubit ActOnArgs, which is part of why I'd expect it to have distinct behavior. Thanks for the fix!